src/components/SubCollapse.tsx

```diff
@@ -0,0 +1,42 @@
+import { GrafanaTheme } from '@grafana/data';
+import { Icon, useStyles } from '@grafana/ui';
+import React, { FC, useState } from 'react';
+import { css } from 'emotion';
+
+interface Props {
+  title: string;
+}
+
+const getStyles = (theme: GrafanaTheme) => ({
+  header: css`
+    border-top: 1px solid #343b40;
```

Comment:
Hard-coded color values always make me wonder... couldn't a shade of grey from the theme be used?
Plus, does this work well with light theme?
Sorry if that's obvious but you know - I've forgotten to polish such things here and there myself. :)